.codeclimate.yml

```diff
@@ -0,0 +1,9 @@
+languages:
+  Ruby: false
```

Comment:
It's not necessary. I just kind of liked to see which options are available :wink: 

If you prefer, I can remove the other languages.